### PR TITLE
feat: licenses screen design update

### PR DESCRIPTION
**Pull request by MohamadJaara**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [ ] contains a reference JIRA issue number like `SQPIT-764`
    - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

licenses screen design did not match the rest of the app

### Solutions

change the UI look like the rest of the app

[Screen_recording_20230704_154929.webm](https://github.com/wireapp/wire-android-reloaded/assets/9083456/5f56d2ac-21f0-4d6b-971b-80add7ce50f6)

### Dependencies (Optional)

_If there are some other pull requests related to this one (e.g. new releases of frameworks), specify them here._

Needs releases with:

- [ ] GitHub link to other pull request

#### Test Coverage (Optional)

- [ ] I have added automated test to this contribution

#### How to Test

_Briefly describe how this change was tested and if applicable the exact steps taken to verify that it works as expected._

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_ 

----
#### PR Post Submission Checklist for internal contributors (Optional)

- [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

- [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [198](https://10.10.124.134/job/AR-build-pipeline/198/) **failed**.

**Comment by AndroidBob** [comment]
Build [200](https://10.10.124.134/job/AR-build-pipeline/200/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.3.0(6951517)(6951517)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.3.0(6951517)(6951517)-staging-compat.apk)

**Review by alexandreferris — APPROVED** [review]

**Comment by AndroidBob** [comment]
Build [208](https://10.10.124.134/job/AR-build-pipeline/208/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.3.0(6959136)(6959136)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.3.0(6959136)(6959136)-staging-compat.apk)

File: app/src/main/kotlin/com/wire/android/ui/home/settings/about/licenses/LicensesItem.kt [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+            library.name,
+            library.author
+        ) {
+            onLibraryClick.invoke(library)
+        }
+    }
+}
+
+@Composable
+fun libItem(
```
Comment (ohassine) [inline app/src/main/kotlin/com/wire/android/ui/home/settings/about/licenses/LicensesItem.kt]: Compostable name should start with uppercase

File: app/src/main/kotlin/com/wire/android/ui/home/settings/about/licenses/LicensesScreen.kt [hunk 2]
```diff
@@ -55,9 +64,34 @@ fun LicensesContent(
...
         ) {
-            LibrariesContainer(
-                modifier = Modifier.fillMaxSize()
+            val context = LocalContext.current
+            val libraries = produceState<Libs?>(null) {
+                value = withContext(Dispatchers.IO) {
+                    Libs.Builder().withContext(context).build()
+                }
+            }
+            val libs = libraries.value?.libraries
```
Comment (ohassine) [inline app/src/main/kotlin/com/wire/android/ui/home/settings/about/licenses/LicensesScreen.kt]: I think this logic should be done in viewModel and then we inject the list of libs.

`LicensesContent` looks like it's `LicensesScreen` we have everyting there. What about putting the scaffold in `LicensesScreen` and `LicensesContent` will just have the content as the names says ?

Comment (MohamadJaara) [reply app/src/main/kotlin/com/wire/android/ui/home/settings/about/licenses/LicensesScreen.kt]: make sense

**Comment by AndroidBob** [comment]
Build [220](https://10.10.124.134/job/AR-build-pipeline/220/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.3.0(6960060)(6960060)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.3.0(6960060)(6960060)-staging-compat.apk)

**Review by ohassine — APPROVED** [review]

**Review by vitorhugods — APPROVED** [review]

**Comment by AndroidBob** [comment]
Build [265](https://10.10.124.134/job/AR-build-pipeline/265/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.3.0(6976792)(6976792)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.3.0(6976792)(6976792)-staging-compat.apk)

**Pull request merged**